### PR TITLE
Update hello_world.md

### PR DESCRIPTION
**Pull request by GS76**
Replacing the original link to the Windows binaries with the following link:

http://www.equation.com/servlet/equation.cmd?fa=fortran

### CODE REVIEW

File: learn/quickstart/hello_world.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ which is part of the
 
 To install gfortran on Linux, use your system package manager.
 Otherwise, for macOS or Windows, refer to gfortran binaries from 
-[this page](https://gcc.gnu.org/install/binaries.html).
+[this page](http://www.equation.com/servlet/equation.cmd?fa=fortran).
```
Comment (milancurcic) [inline learn/quickstart/hello_world.md:25]: The equation.com links seems to serve only Windows binaries. This suggestion adds resources for macOS.

```suggestion
On macOS, you can install gfortran using [Homebrew](https://brew.sh/) or [MacPorts](https://www.macports.org/).
On Windows, you can get native binaries [here](http://www.equation.com/servlet/equation.cmd?fa=fortran).
```

Comment (GS76) [reply learn/quickstart/hello_world.md:25]: @milancurcic,

Thank you for including the macOS in this tutorial. Do we need someone to test this before committing? I do not have a means of testing the macOS setup.

Thanks.

**Comment by GS76** [comment]
@LKedward

I was able to install the gfortran.

I then checked the version in a different path and it worked.

I did not need to define the environment variables.

Regards.

**Comment by GS76** [comment]
Hi all,

I am unsure what I should do now as I do not see this change live on fortran-lang.org?

**Comment by LKedward** [comment]
Hi @GS76, we haven't merged this yet hence why it isn't on the site.
Everything looks good to merge except I notice you haven't committed Milan's suggestion for macOS - if you've done this locally on your computer, don't forget to push again to the same branch. Otherwise you can also commit the change via the web interface.

**Comment by GS76** [comment]
> Hi @GS76, we haven't merged this yet hence why it isn't on the site.
> Everything looks good to merge except I notice you haven't committed Milan's suggestion for macOS - if you've done this locally on your computer, don't forget to push again to the same branch. Otherwise, you can also commit the change via the web interface.

Hi @LKedward 

How are you?

I do not have access to a macOS machine so I can not test this.

Thank you.

**Comment by GS76** [comment]
Hi @LKedward 

Apologies, did I do the wrong thing by closing this as it is unmerged.

I am inexperienced with Github.

How can I correct this

**Comment by LKedward** [comment]
> Apologies, did I do the wrong thing by closing this as it is unmerged.

Not to worry, PR is reopened now.

> I am inexperienced with Github.
> 

git and github have a bit of a learning curve but I promise it doesn't take long to become familiar with the processes. We really appreciate the effort on your part in learning these tools and contributing to the site :+1: 

> ...
> How can I correct this

You won't have permission to merge this pull request yourself - I hope you can understand my reluctance to merge it since, as Milan pointed out, the text currently points macOS users to the site for windows binaries.

> I do not have access to a macOS machine so I can not test this.

I agree this is important and, while I also cannot test this, Milan's suggestion for macOS is exactly what is recommended on the gcc binaries page: <https://gcc.gnu.org/install/binaries.html>.
Finally, I have confidence in the Homebrew system since it is currently used by the [stdlib continuous integration tests](https://github.com/fortran-lang/stdlib/runs/774305181?check_suite_focus=true) (which run after each commit).

**Comment by GS76** [comment]
> > Apologies, did I do the wrong thing by closing this as it is unmerged.
> 
> Not to worry, PR is reopened now.
> 
> > I am inexperienced with Github.
> 
> git and github have a bit of a learning curve but I promise it doesn't take long to become familiar with the processes. We really appreciate the effort on your part in learning these tools and contributing to the site 👍
> 
> > ...
> > How can I correct this
> 
> You won't have permission to merge this pull request yourself - I hope you can understand my reluctance to merge it since, as Milan pointed out, the text currently points macOS users to the site for windows binaries.
> 
> > I do not have access to a macOS machine so I can not test this.
> 
> I agree this is important and, while I also cannot test this, Milan's suggestion for macOS is exactly what is recommended on the gcc binaries page: https://gcc.gnu.org/install/binaries.html.
> Finally, I have confidence in the Homebrew system since it is currently used by the [stdlib continuous integration tests](https://github.com/fortran-lang/stdlib/runs/774305181?check_suite_focus=true) (which run after each commit).

@LKedward 

I am in complete agreement.

We should do it once correctly.

That macOS project looks like the right way to go. Now to just find an Apple user to help us test.

**Comment by LKedward** [comment]
Thanks @GS76.
To be clear, I think this PR can be merged __with Milan's suggestion included__, since the site already recommends these macOS solutions indirectly by referencing the link to gcc binaries page.
However if you would prefer to hold off until they have been tested by the community, then I have no objection to that.
What do you think @milancurcic?

**Comment by milancurcic** [comment]
Sorry @GS76 and @LKedward for not responding sooner to this.

@zbeekman and @scivision can either of you confirm that the following instruction is accurate:

> On macOS, you can install gfortran using [Homebrew](https://brew.sh/) or [MacPorts](https://www.macports.org/).

**Comment by scivision** [comment]
Yes I tell everyone to use Homebrew for Mac OS. I think we should note that the user "brew install gcc" with homebrew

**Comment by GS76** [comment]
Thank you @milancurcic, @LKedward and @scivision.

Let me know how I can help to close this out.

I will then continue testing the tutorials on the site.

File: learn/quickstart/hello_world.md (hunk 1 above)
Comment (milancurcic) [inline learn/quickstart/hello_world.md:25]:
```suggestion
On macOS, you can install gfortran using [Homebrew](https://brew.sh/) or [MacPorts](https://www.macports.org/).
On Windows, you can get native binaries [here](http://www.equation.com/servlet/equation.cmd?fa=fortran).
```

**Comment by scivision** [comment]
The issue is that if users do `brew install gfortran` they're getting not the latest binaries. The main package is `brew install gcc`

**Comment by milancurcic** [comment]
Thank you @scivision. We should definitely add the specific Homebrew instructions, though this belongs to a dedicated section on how to install gfortran, with subsections for Homebrew, Ubuntu, Fedora, Windows, etc. Let's tackle these in a separate PR.

Thank you @GS76, I will merge now.

**Comment by GS76** [comment]
Hi all,

Is this complete? What is remaining now? Sorry, new to Github?

**Comment by milancurcic** [comment]
Yes, it's merged (which automatically closes the PR) and your changes are live on the website. 🎉

**Comment by LKedward** [comment]
Hi @GS76, yes this has been merged and published successfully! Many thanks :+1: 
(Note that the site can take some time to update - see the CONTRIBUTOR guide for how to force a  refresh)

**Pull request merged**